config/config.go

```diff
@@ -71,55 +70,48 @@ type GlobalFlagValues struct {
 	NoColor          bool
 }
 
-func DefaultConfig() *Config {
+func DefaultConfig() Config {
 	hz := hazelcast.Config{}
 	hz.Cluster.Unisocket = true
 	hz.Logger.Level = logger.ErrorLevel
 	hz.Cluster.Name = DefaultClusterName
 	hz.Stats.Enabled = true
-	return &Config{Hazelcast: hz}
+	return Config{Hazelcast: hz}
 }
 
-const defaultUserConfig = `hazelcast:
-  clientname: ""
-  logger:
-    level: error
+const defaultUserConfig = `
+hazelcast:
   cluster:
-    security:
-      credentials:
-        username: ""
-        password: ""
     name: dev
+    unisocket: true
+    network:
+      # 0s is no timeout
+      connectiontimeout: 0s
+      addresses:
+      - "localhost:5701"
     cloud:
       token: ""
       enabled: false
+    security:
+      credentials:
+        username: ""
+        password: ""
     discovery:
       usepublicip: false
-    unisocket: true
-  network:
-    addresses:
-      - "localhost:5701"
-    # 0s means infinite timeout (no timeout)
-    connectiontimeout: 0s
+  logger:
+    level: error
 ssl:
   enabled: false
   servername: ""
   capath: ""
   certpath: ""
   keypath: ""
   keypassword: ""
-# disables auto completion on interactive mode
+# disables auto completion on interactive mode if true
 noautocompletion: false
 styling:
-  theme: "default" # default, no-color, solarized
-  colorpalette:
-    # uncomment to override theme color. closest supported color will be used
-    #headerbackground: "#ff12aa"
-    #border: "#ff12aa"
-    #resulttext: "#ff12aa"
-    #headerforeground: "#000000"
-    #highlight: "#ff12aa"
-    #footerforeground: "#ff12aa"
```

Comment:
I'm assuming we are removing the color options intentionally.

Reply:
The color options are removed only from the default configuration to make the default config contain the most important options. The user can still add and use them.